src/actions/etherspotActions.js

```diff
@@ -283,65 +279,21 @@ export const fetchDefaultTokens = () => {
 
     const supportedChain = getSupportedChains(activeAccount);
 
-    dispatch({ type: FETCHING_DEFAULT_LIST, payload: true });
-
-    // For Get Default Token List
-    const chainTokens = await Promise.all(
-      supportedChain?.map((chain) => etherspotService.getTokenListTokens(chain, ETHERSPOT_DEFAULT_LIST)),
-    ).catch((e) => {
-      reportErrorLog('EtherspotActions fetchDefaultTokens -> getTokenListTokens failed', {
-        error: e,
-        name: ETHERSPOT_DEFAULT_LIST,
-      });
-      return [];
-    });
+    await Promise.all(
+      supportedChain?.map(async (chain: Chain) => {
+        const tokens = DefaultTokens.concat(DefaultStableTokens);
 
-    // For Get Default Stable List
-    const chainStableTokens = await Promise.all(
-      supportedChain?.map((chain) => etherspotService.getTokenListTokens(chain, ETHERSPOT_DEFAULT_STABLE_LIST)),
-    ).catch((e) => {
-      reportErrorLog('EtherspotActions fetchDefaultTokens -> getTokenListTokens failed', {
-        error: e,
-        name: ETHERSPOT_DEFAULT_STABLE_LIST,
-      });
-      return [];
-    });
+        const chainAssets: Asset[] = filteredWithChain(tokens, chain);
 
-    const tokens: any[] = [];
-    await Promise.all(
-      chainTokens?.map(async (item: any) => {
-        if (!isEmpty(item)) {
-          tokens.push(...item);
-          await getExchangeRates(item[0].chain, item, async (rates) => {
-            if (rates) await dispatch(updateRatesAction(item[0].chain, rates));
-          });
-        }
+        await getExchangeRates(chain, chainAssets, async (rates) => {
```

Comment:
What happens if one of these rates fail?

Reply:
Call catch fn

```
} catch (error) {
          reportErrorLog('Fetch Rates failed: request error', {
            error,
            assetsContractAddresses,
          });
        }
```